FirebaseCore/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Firebase 8.0.0
```

Comment:
Do you want to also mention removal of deprecated pods from the header?

Reply:
Done. PTAL.